packages/fluentui/docs/src/components/ComponentDoc/ComponentControls/ComponentControls.tsx

```diff
@@ -33,7 +33,7 @@ type ComponentControlsProps = {
   showRtl: boolean;
   showVariables: boolean;
   showTransparent: boolean;
-  toolbarAriaLabel?: string;
+  title?: React.ReactNode;
```

Comment:
do we somewhere use something else than `string` for `title`?

Reply:
changed to string :)